libobs/obs-video.c

```diff
@@ -728,27 +814,52 @@ static inline void output_frame(bool raw_active, const bool gpu_active)
 	int cur_texture = video->cur_texture;
 	int prev_texture = cur_texture == 0 ? NUM_TEXTURES - 1
 					    : cur_texture - 1;
-	struct video_data frame;
+	struct video_data main_frame;
+	struct video_data streaming_frame;
+	struct video_data recording_frame;
 	bool frame_ready = 0;
 
-	memset(&frame, 0, sizeof(struct video_data));
+	memset(&main_frame, 0, sizeof(struct video_data));
```

Comment:
memset can be replaced with struct initialization to zero like  **= {0}** ?

Reply:
Done 👍